manifests/utility/database.pp

```diff
@@ -0,0 +1,24 @@
+# Sample Usage:
+#   mariadb::utility::database { "production dbname" :
+#       dbname => 'dbname',
+#       charset => 'utf8',
+#       collate => 'utf8_general_ci'
+#       ensure => 'present',
+#   }
+define mariadb::utility::database($dbname, $charset = 'utf8', $collate = 'utf8_general_ci', $ensure = 'present' ) {
```

Comment:
We take in hashes already to pass on to the underlying mysql module here: https://github.com/edestecd/puppet-mariadb/blob/master/manifests/cluster.pp#L78
I think that solves the "one liner", or less code approach. More documentation would be nice, as maybe thats not clear.

I worry about maintaining such wrappers. Every time the underlying types change in the mysql module, we would be constantly updating parameters to our wrappers. I think that is harder to maintain. You don't have that problem passing through hashes. I think just using the underlying types as they are and contributing to them in that module is the best approach. I do not wish to provide that functionality as I think the mysql module does it well.